sdks/python/apache_beam/ml/inference/onnx_inference.py

```diff
@@ -116,7 +116,16 @@ def load_model(self) -> ort.InferenceSession:
     # when path is remote, we should first load into memory then deserialize
     f = FileSystems.open(self._model_uri, "rb")
     model_proto = onnx.load(f)
-    model_proto_bytes = onnx._serialize(model_proto)
+    model_proto_bytes = model_proto
+    if not isinstance(model_proto, bytes):
```

Comment:
model_proto_bytes should be used below, right?

Reply:
Yeah, good catch - fixed